packages/components/src/components/linear-progress/linear-progress-container.jsx

```diff
@@ -29,6 +31,12 @@ const LinearProgressContainer = ({ timeout, action, render, className }) => {
         }
     });
 
+    if (sessionStorage.getItem('popup_timer') <= 0) {
+        sessionStorage.removeItem('popup_timer');
+    } else {
+        sessionStorage.setItem('popup_timer', timeout_state);
```

Comment:
Looks like this will share the same session for multiple `LinearProgress`. If there are multiple `LinearProgress` component mounted, then the same `popup_timer` session value will be overridden from multiple components.

We should be tracking the timeout using a unique id. 
Also, please introduce a boolean prop in `LinearProgress` component to store the state in the session as we don't need if for all usages.

Reply:
I pass now a unique ID into the component that gets assigned to this components' timeout session. Now, every instance of this component will have a unique timer.

A boolean has also been set in the props.